packages/xo-web/src/common/render-xo-item.js

```diff
@@ -697,8 +706,57 @@ const xoItemToRender = {
     )
   },
   job: job => <spans>{job.name}</spans>,
+
+  backupJob: backupJob => {
+    const labels = []
+
+    if (isFullBackup(backupJob)) {
+      labels.push(_('fullBackup'))
+    } else if (isDeltaBackup(backupJob)) {
+      labels.push(_('incrementalBackup'))
+    }
+
+    if (isDrBackup(backupJob)) {
+      labels.push(_('fullReplication'))
+    } else if (isCrBackup(backupJob)) {
+      labels.push(_('incrementalReplication'))
+    }
+
+    if (isRollingSnapshotBackup(backupJob)) {
+      labels.push(_('rollingSnapshot'))
+    }
+
+    return <BackupJobSelect backupJob={backupJob} labels={labels} />
+  },
+  metadataBackup: backupJob => {
+    const labels = []
+    if (isPoolMetadataBackup(backupJob)) {
+      labels.push(_('poolMetadata'))
+    }
+    if (isXoConfigBackup(backupJob)) {
+      labels.push(_('xoConfig'))
+    }
+    return <BackupJobSelect backupJob={backupJob} labels={labels} />
+  },
+  mirrorBackup: backupJob => (
+    <BackupJobSelect
+      backupJob={backupJob}
+      labels={[_(backupJob.mode === 'delta' ? 'mirrorIncrementalBackup' : 'mirrorFullBackup')]}
+    />
+  ),
 }
 
+const BackupJobSelect = ({ backupJob, labels }) => (
```

Comment:
Why "Select"?

Reply:
1. Because `BackupJob` already exists in this file.
2. Because it is the representation of a `BackupJob` inside a `Select` component

Reply:
1. Then maybe we can re-use `BackupJob` instead? With an option to show labels?
2. If the answer to 1. is "no", then let's call this one `BackupJobOption`